install_config/install/advanced_install.adoc

```diff
@@ -1362,10 +1362,53 @@ specifications, and save it as *_/etc/ansible/hosts_*.
 [[running-the-advanced-installation]]
 == Running the Advanced Installation
 
-After you have xref:configuring-ansible[configured Ansible] by defining an
-inventory file in *_/etc/ansible/hosts_*, you can run the advanced installation
-using the following playbook:
+After you have finished xref:configuring-ansible[configuring Ansible] by
+defining your own inventory file in *_/etc/ansible/hosts_* or modifying one of
+the xref:adv-install-example-inventory-files[example inventories], follow these
+steps to run the advanced installation:
 
+// tag::BZ1466783-workaround-install[]
+. In {product-title}
+ifdef::openshift-enterprise[]
+3.4,
+endif::[]
+ifdef::openshift-origin[]
+1.4,
+endif::[]
+the master connected to the etcd cluster using the host name of the etcd
+endpoint. In {product-title}
+ifdef::openshift-enterprise[]
+3.5,
+endif::[]
+ifdef::openshift-origin[]
+1.5,
+endif::[]
+the master now connects to etcd via IP address.
++
+When configuring a cluster to use proxy settings (see
+xref:advanced-install-configuring-global-proxy[Configuring Global Proxy Options]), this change causes the master to etcd connection to be proxied as
+well, rather than being excluded by host name in each host's `NO_PROXY` setting
+(see
+xref:../../install_config/http_proxies.adoc#install-config-http-proxies[Working with HTTP Proxies] for more about `NO_PROXY`).
++
+If you are not using a proxy, you can skip this step. Otherwise, to workaround
+this issue, add the IP address of the etcd endpoint to the `openshift_no_proxy`
+cluster variable in your inventory file. For example:
++
+----
+openshift_no_proxy=https://<ip_address>:<port>
+----
++
+Use the IP that the master will use to contact the etcd cluster as the
+`<ip_address>`. The `<port>` should be `2379` if you are using standalone etcd
+(clustered) or `4001` for embedded etcd (single master, non-clustered etcd). The
+installer will be updated in a future release to handle this scenario
+automatically during installation and upgrades
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1466783[*BZ#1466783*]).
+// end::BZ1466783-workaround-install[]
+
+. Finally, run the advanced installation using the following playbook:
```

Comment:
Probably don't need "Finally" here. Just "Run the advanced install..... "
(Finally, i can install the darn thing!)